source/index.html.md.erb

```diff
@@ -8,18 +8,65 @@ title: Reliability Engineering
 
 ## Services we provide
 
-### Logging
+Reliabilty Engineering provide a shared platform to GDS teams with the tools you need to set up and maintain a service, we do this by:
 
-We provide ELK (elasticsearch/logstash/kibana) stacks via
+* acquiring tools and where appropriate administer them for GDS, for example Logit
+* running off-the-shelf services for GDS as internal SaaS, for example Prometheus
+* providing patterns and guidance, for example PaaS incident templates
+
+To understand the context for our decisions and guidance you'll need to refer to the [Service Manual](https://www.gov.uk/service-manual) and [GDS Way](https://gds-way.cloudapps.digital/#gds-technical-guidance).
+
+### Logit
+
+You can use Logit as your logging solution, it's been approved for use at GDS.
+
+We provide ELK (elasticsearch/logstash/kibana) stacks using
 [logit](https://logit.io).  We have guides to help you:
 
 - [Get a logit account](manuals/logit-io-joiners-and-leavers.html)
-- Ship logs from S3 to logit (TODO)
+- Ship logs from S3 to logit (guidance in progress)
 - [Send logs securely to Logit.io](manuals/logit-io.html)
 
 The PaaS docs document
 [how to send logs from PaaS to logit](https://docs.cloud.service.gov.uk/#set-up-the-logit-io-log-management-service).
 
+### Confluence
+
+[Confluence](https://www.atlassian.com/software/confluence) is an approved tool for GDS service teams to use.
```

Comment:
What does "approved" mean? Do we have a hosted solution? Are teams ok to procure their own solution? What actions can a team take with this information?

Reply:
Good point. We shouldn't be using opaque language like this. If it's been through IA and procured then it could read, `has been approved by GDS information assurance and procured for use by all GDS teams`.

Reply:
Alternatively you could consider IA approval to be implicit (we probably should) and just write `has been procured for use by all GDS teams`.

Reply:
Agreed - have changed this